server/src/server.ts

```diff
@@ -0,0 +1,55 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
+import fs from "fs";
+import { parse, ParsedQuery } from "query-string";
+
+const port = 4000;
+
+const isPerson = (obj: any): boolean => {
+  return obj.name.length > 0 && Date.parse(obj.date) > 0;

```

Comment:
Will be enough to check obj.name. https://stackoverflow.com/questions/43309712/how-to-check-if-a-value-is-not-null-and-not-empty-string-in-js/64186744

Are you hate people older than 01.01.1970? =)
Date.parse('01.01.1960')

Reply:
Removed unnecessary data validation